ReactiveCocoa/Swift/Signal.swift

```diff
@@ -128,8 +128,8 @@ public final class Signal<Value, Error: ErrorType> {
 		}
 
 		if let token = token {
-			return ActionDisposable {
-				self.atomicObservers.modify { observers in
+			return ActionDisposable { [weak self] in
```

Comment:
No, something in ActionSpec.swift crashes. Apparently, disposable can be disposed after the signal is deallocated.